### ISSUE TITLE
New message indicator appears when there are no new messages

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#799]
I don't know what causes this to happen, but there are cases when the purple "new messages" indicator displays again and again (intermittently, seemingly at random) when there are not any new messages in a channel.

For me it's happening in channels that have been dormant for weeks.

**Comment by kingalg** [comment]
@holmesworcester does it still happen? Because we were fixing some things with new message indicator in a last release and it's possible that it was fixed while we were handling other tasks. I never encounter it even in our long running community.

**Comment by Kacper-RF** [comment]
Yes, I believe it was a few weeks ago, but it was probably a duplicate issue that was moved to done.
So the issue is not finally resolved and you are still seeing this problem?

**Comment by holmesworcester** [comment]
I don't see it happening now so let's close

**Comment by holmesworcester** [comment]
I'm still seeing it in Quiet 1.9.0 in Ubuntu. I don't know how to reproduce it but it seems like it happens when I send a message with a link

![image](https://github.com/TryQuiet/quiet/assets/213678/a9079f7c-8d7d-44bd-b94f-076c82fc7f50)
